OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -461,12 +463,15 @@ void UpdateCellBlocking(CPos cell)
 								actorBits = actorBits.Union(crushingBits);
 								cellBlocking |= CellBlocking.Crushable;
 							}
-
-						if (isMoving)
-							actorBits = actorBits.Except(actor.Owner.AllyMask);
 					}
 					else
-						actorBits = isMoving ? actor.Owner.EnemyMask : allPlayerMask;
+						actorBits = allPlayerMask;
+
+					if (isMoving)
+					{
+						actorBits = actorBits.Except(actor.Owner.AllyMask);
```

Comment:
We can only ignore moving allies when `!check.HasCellCondition(CellConditions.BlockedByMovers)`